### ISSUE TITLE
Accessibility question about JavaScript

### ISSUE THREAD

**Issue by jsguy1** [wet-boew/wet-boew#8525]
We have a complex form that sends data to a third-party. The functionality of sending the data from A to B relies on JavaScript. The form interface/controls are WCAG 2.0 accessible, but if the user happens to have JS disabled, they are brought to a page with a <noscript> tag that tells them that the site requires Javascript.

I am well aware that this is not ideal, and that progressive enhancement techniques should be used.  However, let's just say that it took me a lot of blood, sweat and tears to finally get the form working.

From a rules perspective, can a form that relies on JavaScript to transmit the form data still meet WCAG 2.0 requirements?

**Comment by duboisp** [comment]
> Can a form that requires JavaScript still meet WCAG 2.0 requirements?

Yes, we consider JavaScript being [accessibility supported](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html#accessibility-supporteddef).

Using progressive enhancement technique for web accessibility are a best practice and not a requirement to being conform to WCAG 2.0 Level AA.

**Comment by jsguy1** [comment]
Thanks!
